core/src/replay_stage.rs

```diff
@@ -2472,6 +2472,28 @@ impl ReplayStage {
 
             assert_eq!(bank_slot, bank.slot());
             if bank.is_complete() {
+                let r = bank.wait_for_scheduler();
```

Comment:
ooo this is good idea, im guessing they run in background? might have to take that one into mine lol